apps/web/components/Organization/Slice/TeamListSlice/TeamListSlice.tsx

```diff
@@ -132,7 +95,7 @@ export const TeamMemberListWrapper = ({
       tagQueryId={tagQueryId}
     >
       <TeamList
-        teamMembers={items}
+        teamMembers={items as TeamListProps['teamMembers']}
```

Comment:
Can we skip using "as" here and do the type definition where the "items" is initialized?